src/coordinate.zig

```diff
@@ -497,6 +497,9 @@ fn parseDependenciesFromPackageRoot(
             .start = 0,
             .len = 0,
         } },
+        .malformed => {
+            @panic("TODO -- what should we do for a malformed node here?? raise a compiler problem I assume");
```

Comment:
Long term, it sounds like we want to be able to skip over the node. So just copy it verbatim and then keep formatting everything else.

Sort term, this should be unreachable. We should ensure there are no parse errors before formatting.

Reply:
Sorry, meant to write this in `fmt.zig`. Not sure for coordinate. Probably push a compiler problem (though one should already exit from parse?) and ignore it if possible.